examples/ex_invariant_so3.py

```diff
@@ -31,9 +38,9 @@
     [mag_model, grav_model],
     1,
 )
-state_true, input_list, meas_list = dg.generate(x0, 0, 30, True)
-
-
+state_true, input_list, meas_list = dg.generate(x0, 0, 30, noise_active)
+x0.plus(randvec(P0))
```

Comment:
If noise_active == False, no initial condition shouldnt be corrupted right?
```suggestion
if noise_active:
    x0.plus(randvec(P0))
```